llvm/tools/llvm-profgen/ProfiledBinary.cpp

```diff
@@ -37,6 +37,13 @@ cl::opt<bool> ShowSourceLocations("show-source-locations",
                                   cl::desc("Print source locations."),
                                   cl::cat(ProfGenCategory));
 
+cl::opt<bool>
+    LoadFunctionFromSymbol("load-function-from-symbol", cl::init(true),
```

Comment:
We probably need a better name for this flag. The function name can come from either the symbol table or pseudo probes.